components/molecule/collapsible/src/index.js

```diff
@@ -10,10 +10,15 @@ const BUTTON_CLASS = `${BASE_CLASS}-btn`
 const BUTTON_CONTENT_CLASS = `${BUTTON_CLASS}-content`
 const ICON_CLASS = `${BASE_CLASS}-icon`
 const MIN_HEIGHT = 100 // px
+const CONTENT_ALIGN = {
```

Comment:
you can expose this value out of the component

```javascript

export {CONTENT_ALIGN}
```
...so then you'll be able to use it:

```javascript

import MoleculeCollapsible, {CONTENT_ALIGN} from ' @s-ui/react-molecule-collapsible' 
```